parsec/backend/postgresql/organization.py

```diff
@@ -438,7 +443,10 @@ async def update(
         if with_is_expired:
             fields["is_expired"] = is_expired
         if with_active_users_limit:
-            fields["active_users_limit"] = active_users_limit
+            assert isinstance(active_users_limit, ActiveUsersLimit)
+            fields["active_users_limit"] = (
+                active_users_limit if active_users_limit != ActiveUsersLimit.NO_LIMIT else None
+            )
```

Comment:
use `to_int` ?

Reply:
I wanted to make it clear that `NO_LIMIT` is `NULL` in the pgsql database. What do you think?

Reply:
Yes, a call to `to_int` would work here